### PR TITLE
fix: coinbase wallet not showing qr code

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
# Why
Tapping coinbase wallet in rainbowkit modal does not show QR code.

# How
- Coinbase internally uses `eth-block-tracker` during connecting wallet but it was not working as yarn resolution was forcing it to use another version.
- Remove `eth-block-tracker` from yarn resolutions.

# Test Plan
Tapping coinbase should show QR code

### CODE REVIEW

File: apps/next/next.config.js [hunk 1]
```diff
@@ -76,6 +76,7 @@ const nextConfig = {
     "react-native-reanimated-carousel",
   ],
   webpack: (config, options) => {
+    config.resolve.fallback = { fs: false, net: false, tls: false };
```
Comment (intergalacticspacehighway) [inline apps/next/next.config.js]: this is recommended in latest rainbowkit as it doesn't polyfill these deps anymore

Comment (hirbod) [reply apps/next/next.config.js]: This is not save enough

Comment (hirbod) [inline apps/next/next.config.js]:
```suggestion
    if (!options.isServer) {
        config.resolve.fallback = { fs: false, net: false, tls: false };
    }
```

**Review by hirbod — CHANGES_REQUESTED** [review]

**Review by hirbod — APPROVED** [review]
LGTM

**Pull request merged**